src/main/java/com/fleotadezuta/youthprogrammanager/model/UserDetails.java

```diff
@@ -0,0 +1,16 @@
+package com.fleotadezuta.youthprogrammanager.model;
+
+import graphql.GraphQLContext;
+import lombok.Getter;
+
+@Getter
+public class UserDetails {
+    private final String userId;
+    private final String userType;
+
+    public UserDetails(GraphQLContext context) {
+        this.userId = context.get("X-app-user-id").toString();
+        this.userType = context.get("X-app-user-type").toString();
```

Comment:
Ahogy fentebb is írtam, jöjjön ez a két érték a megfelelő konstansokat tartalmazó osztályokból.

Reply:
Továbbra is igaz.